app/controllers/comments_controller.rb

```diff
@@ -5,6 +5,13 @@ def create
     redirect_to article_path(@article)
   end
 
+  def destroy
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.find(params[:id])
+    @comment.destroy
```

Comment:
In general, if you don't need to access the variable from other method or view in the controller, you don't have to set an instance variable, it could be a local variable simply.

In this case, the method `destroy` could take an id and do the finding by its own.
https://apidock.com/rails/ActiveRecord/Relation/destroy

```suggestion
    @article.comments.destroy(params[:id])
```